src/pudl/etl/cli.py

```diff
@@ -23,42 +19,6 @@
 logger = pudl.logging_helpers.get_logger(__name__)
 
 
-def pudl_etl_job_factory(
```

Comment:
This was only being used in `conftest` so I moved it over to avoid some import issues.